ironfish-cli/src/commands/wallet/post.ts

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+
+export class PostCommand extends IronfishCommand {
+  static description = `Post a raw transaction`
+
+  static flags = {
+    ...RemoteFlags,
+  }
+
+  static args = [
+    {
+      name: 'transaction',
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      description: 'The raw transaction in hex encoding',
+    },
+  ]
+
+  async start(): Promise<void> {
+    const { args } = await this.parse(PostCommand)
+    const transaction = args.transaction as string
+
+    CliUx.ux.action.start(`Posting transaction`)
```

Comment:
Can you deserialize the raw transaction, display the details and confirm with user they want to sign this transaction

Reply:
Yep, I'll add that

Reply:
Added